azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/redux/action/PermissionAction.kt

```diff
@@ -9,6 +9,8 @@ internal sealed class PermissionAction :
     Action {
     class AudioPermissionRequested : PermissionAction()
     class CameraPermissionRequested : PermissionAction()
+    class PhonePermissionRequested : PermissionAction()
```

Comment:
These are all stateless, can we convert them to sealed objects instead

Reply:
Converted